### DISCUSSION TITLE
Avoid using Renovate as a verb

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#23405] [Suggest an Idea]
### Tell us more.

https://github.com/renovatebot/renovate/blob/e75ff54abb273ae28d5ff655c9f2b9c4e98292c0/lib/workers/repository/dependency-dashboard.ts#L171-L185

We use the Renovate name as a verb: `renovating` on line 178. I'd like use to use a plain English verb instead. And also get rid of the boring `occurred`. 😄 

What do you think of: Renovate tried to update this repository, but found these problems.

**Comment by HonkingGoose** [comment]
I'm resolving this Discussion with this PR:

- #23408

**Reply by jamietanna** [reply]
And a follow-up for some new findings: https://github.com/renovatebot/renovate/pull/40051
